src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -7825,6 +7860,192 @@ private static int Test(int x, out int y)
                 );
         }
 
+        [Fact]
+        public void BaseArguments_19()
+        {
+            var src = @"
+record Base
+{
+    public Base(int X)
+    {
+    }
+
+    public Base() {}
+}
+
+record C(int X, int Y) : Base(GetInt(X, out var xx) + xx, Y), I
```

Comment:
>Base [](start = 25, length = 4)

This should also be tested with a class (not record) as the base, as permitted by [https://github.com/dotnet/csharplang/blob/master/proposals/records.md](https://github.com/dotnet/csharplang/blob/master/proposals/records.md)

Reply:
> This should also be tested with a class (not record) as the base, as permitted by [https://github.com/dotnet/csharplang/blob/master/proposals/records.md](https://github.com/dotnet/csharplang/blob/master/proposals/records.md)

Any particular aspect you have in mind? There we some existing tests with classes with the old representation.

---
In reply to: [443928983](https://github.com/dotnet/roslyn/pull/45351#discussion_r443928983) [](ancestors = 443928983)

Reply:
BaseArguments_12 for example.

---
In reply to: [443929594](https://github.com/dotnet/roslyn/pull/45351#discussion_r443929594) [](ancestors = 443929594,443928983)

Reply:
That addresses my concern.

---
In reply to: [443930063](https://github.com/dotnet/roslyn/pull/45351#discussion_r443930063) [](ancestors = 443930063,443929594,443928983)